runelite-client/src/main/java/net/runelite/client/plugins/mining/MiningPlugin.java

```diff
@@ -120,6 +134,10 @@ public class MiningPlugin extends Plugin
 	private final List<RockRespawn> respawns = new ArrayList<>();
 	private boolean recentlyLoggedIn;
 
+	@Getter(AccessLevel.PACKAGE)
+	private final Map<TileObject, Tile> camTorumStreams = new HashMap<>();
```

Comment:
Why is this a `Map<TileObject, Tile>` instead of a `Set<DecorativeObject>`? The only thing the tile is referenced for is its worldpoint, which can be found from `TileObject#getWorldLocation()`.